src/lib/config.ts

```diff
@@ -159,11 +171,11 @@ async function fetchAvatar(
     }
   }
   if (defaultWearableUrns.length > 0) {
-    const defaultWearables = await fetchURNs(defaultWearableUrns, peerUrl)
+    const defaultWearables = await fetchWearableURNs(defaultWearableUrns, peerUrl)
     wearables = [...wearables, ...defaultWearables]
   }
 
-  return wearables
+  return wearables as WearableDefinition[]
```

Comment:
As you're casting what is being returned, can you assign it to the return type of the function? If not, the cast might seem redundant.